src/jsx/modal.jsx

```diff
@@ -11,25 +11,36 @@ class ModalMessage extends Component {
         this.state = {
             showModal: true
         };
-        this.close = this.close.bind(this);
+        this.closeModal = this.closeModal.bind(this);
     }
 
     /**
-     * Closes the modal and performs the action for pressing the cancel button.
+     * Handles key events. If return is pressed, the modal will close, performing the primary function. If escape is
+     * pressed, the secondary function is performed if provided, else the primary function
+     *
+     * @param event
      */
-    close() {
-        this.props.secondaryButtonClicked();
-        this.setState({
-            showModal: false
-        });
+    closeModal(event) {
+        let KEYCODE_ENTER = 13,
```

Comment:
Why not use `const` here? Or move them up to be class constants, even if they are only used in this method.